File: src/Services/BudgetCast.Notifications/src/BudgetCast.Notifications.Hub/Models/NotificationMessageTypes.cs
namespace BudgetCast.Notifications.AppHub.Models
{
    public static class NotificationMessageTypes
    {
        public const string Unknown = nameof(Unknown);

        public const string ExpensesAdded = nameof(ExpensesAdded);
        public const string ExpensesRemoved = nameof(ExpensesRemoved);

        public const string CampaignAdded = nameof(CampaignAdded);
        public const string CampaignJoined = nameof(CampaignJoined);
    }
}

File: src/Web/WebSPA/BudgetCast.Spa/BudgetCast.Spa/Client/src/app/models/notifications/notification-message-types.ts
/**
 * Notification types which might be sent by the notification service.
 */
export enum NotificationMessageTypes {
    Unknown = "Unknown",

    // expenses
    ExpensesAdded = "ExpensesAdded",
    ExprensesRemoved = "ExpensesRemoved",

    // campaigns
    CampaignAdded = "CampaignAdded",
    CampaignJoined = "CampaignJoined"
}